src/RequestLog/Utilities/SecurityUtility.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Cego\RequestLog\Utilities;
+
+use Illuminate\Support\Arr;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+
+class SecurityUtility
+{
+    /**
+     * Get headers of the request with encryption applied according to 'X-SENSITIVE-REQUEST-HEADERS-JSON'
+     *
+     * @param Request $request
+     *
+     * @return string
+     */
+    public static function getHeadersWithMaskingApplied(Request $request): string
+    {
+        $headers = $request->headers->all();
+
+        if ( ! $request->hasHeader('X-SENSITIVE-REQUEST-HEADERS-JSON')) {
+            return json_encode($headers);
+        }
+
+        $sensitiveHeaders = (array) json_decode($request->header('X-SENSITIVE-REQUEST-HEADERS-JSON'));
+
+        $sensitiveHeaders = array_map(fn ($header) => strtolower($header), $sensitiveHeaders);
+
+        foreach ($sensitiveHeaders as $sensitiveHeader) {
+            if (isset($headers[$sensitiveHeader][0])) {
```

Comment:
Tænker det må være muligt at der er flere entries end [0], eftersom det er lavet sådan. Evt. for multiple values for den samme header?